src/common/dateHelpers.js

```diff
@@ -1,6 +1,7 @@
 import moment from 'moment/moment';
 import { helpers } from './helpers';
 import { RHSM_API_QUERY_GRANULARITY_TYPES as GRANULARITY_TYPES } from '../types/rhsmApiTypes';
+import { translate } from '../components/i18n/i18n';
```

Comment:
note:
we used i18next to handle the string translations instead of moment's built in locale handling on purpose. there is a tech debt story to remove moment at some point